domain/src/main/java/org/apache/rocketmq/eventbridge/domain/model/apidestination/ApiDestinationService.java

```diff
@@ -57,7 +58,11 @@ public String createApiDestination(ApiDestinationDTO eventApiDestinationDTO) {
             return null;
         } catch (Exception e) {
             log.error("ApiDestinationService | createApiDestination | error", e);
-            throw new EventBridgeException(e);
+            if (e instanceof EventBridgeException) {
+                EventBridgeException eventBridgeException = (EventBridgeException)e;
+                throw new EventBridgeException(eventBridgeException.getCode(), eventBridgeException.getMessage());
```

Comment:
Move to Exception Handler, remove the exception catch and. duplicate invalid code

Reply:
remove redundant try catch